subprojects/gradle/build-metrics/src/main/kotlin/com/avito/android/plugin/build_metrics/internal/GradleExt.kt

```diff
@@ -1,8 +1,27 @@
 package com.avito.android.plugin.build_metrics.internal
 
+import com.avito.android.gradle.profile.TaskExecution
+import com.avito.android.stats.SeriesName
 import org.gradle.api.internal.GradleInternal
+import org.gradle.api.internal.tasks.TaskStateInternal
 import org.gradle.api.invocation.Gradle
 import org.gradle.internal.operations.BuildOperationListenerManager
+import org.gradle.util.Path
+
+internal fun Path.toSeriesName(): SeriesName {
```

Comment:
The idea is to collect internal extensions for `org.gradle.*` classes.
Otherwise, it will be a file for each receiver class: `PathExt`, `TaskExt`, `GradleExt` and so on.
Did you mean this scheme?